contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM resin/rpi-raspbian:jessie
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/deb/armhf/generate.sh"!
+#
 
-# allow replacing httpredir mirror
-ARG APT_MIRROR=httpredir.debian.org
-RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
+FROM raspbian:jessie
```

Comment:
This one needs an exception as well (alternatively, skip `raspbian` in the script if it's too much different)